modules/security-csrf/src/main/java/com/devonfw/module/security/csrf/service/api/rest/CsrfRestService.java

```diff
@@ -0,0 +1,29 @@
+package com.devonfw.module.security.csrf.service.api.rest;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+
+import org.springframework.security.web.csrf.CsrfToken;
+
+import com.devonfw.module.rest.common.api.RestService;
+
+/**
+ * The CSRF security REST service provides access to the csrf token, the authenticated user's meta-data. Furthermore, it
+ * provides functionality to check permissions and roles of the authenticated user.
+ */
+@Path("/security/v1")
```

Comment:
IMHO we should replace `security` with `csrf`. This is not the "universal generic security service" anymore and that is on purpose as we want to be modular and avoid mixing multiple features that may collide.